tools/ColorTool/ColorTool/IniSchemeParser.cs

```diff
@@ -127,12 +154,33 @@ public ColorScheme ParseScheme(string schemeName, bool reportErrors = false)
 
             if (colorTable != null)
             {
-                return new ColorScheme { colorTable = colorTable };
+                return new ColorScheme { colorTable = colorTable, background = backgroundColor, foreground = forgroundColor, popupBackground = popupBackgroundColor, popupForeground = popupForgroundColor };
```

Comment:
I feel like this is a lot of parameters to add. Can they be put in a struct and passed that way as a single unit?

Reply:
Should that struct ony contain the foreground and background colors or should it also contain the color table?

Reply:
I feel like conceptually the foreground/background stuff all goes together in one struct and the color table is already one. I'm happy with two parameters here. @zadjii-msft, opinion? Am I just being too picky?